lib/rofl_ofdpa_fm_driver.cpp

```diff
@@ -2412,8 +2412,8 @@ cofgroupmod rofl_ofdpa_fm_driver::enable_group_l3_unicast(
   uint32_t group_id = group_id_l3_unicast(id);
   cofgroupmod gm(ofp_version);
 
-  assert(0 == get_group_type(l2_interface) &&
-         "wrong l2 interface in enable_group_l3_unicast");
+  //assert(0 == get_group_type(l2_interface) &&
+         //"wrong l2 interface in enable_group_l3_unicast");
```

Comment:
Why not extend the check to allow trunk interfaces as well instead of removing the check completely?